dts/bindings/usb/usb.yaml

```diff
@@ -0,0 +1,50 @@
+---
+title: USB Base Structure
+id: USB
+version: 0.1
+
+description: >
+    This binding gives the base structures for all USB devices
+
+properties:
+  - compatible:
+      type: string
+      category: required
+      description: compatible strings
+
+  - clock-frequency :
+      type: int
+      category: optional
+      description: Initial clock frequency in Hz
+      generation: define
+
+  - clocks:
+      type: array
+      category: required
+      description: Clock gate information
+      generation: define
+
+  - num-bidir-endpoints:
+      type: int
+      category: required
+      description: Number of bi-directional endpoints (including EP0)
```

Comment:
Maybe say something like:

Number of bi-directional endpoints that the hardware supports (including EP0)

[ same cleanup for num-in-..., num-out-... ]

Reply:
done